manifest/x86_64/z/zed.filelist

```diff
@@ -1,5 +1,4 @@
 /usr/local/bin/zed
-/usr/local/libexec/zed-editor
```

Comment:
Is this needed to run the editor?

Reply:
I don't think so. In the separate linux install it's just zed renamed to zed-editor.

Please try it to see if it works?